### DISCUSSION TITLE
New feature: Browser object in the room - thoughts

### DISCUSSION THREAD

**Discussion by AlbertBaubleDeem** [mozilla/hubs#4029] [General Questions & Discussion] [answered]
**Feature I am missing in Hubs:**

A browser that opens inside the room and users can use it inside the room. A floating window.

The content of the window would be _user specific_ (each user can see his own content in the browser window based on the url inserted by the creator of "the browser window object" and interact with the web page currently loaded).

I am beginning to teach classes in Hubs and this is the one feature I am really missing. I can imagine Google Jamboard opened in a browser window and many other apps.

Having a separate tab opened in my actual browser complicates things as I cannot readily interact with other users while working in an app or showing a presentation etc.

Some of these issues can be solved by window sharing option, but not all.

What do you think?

**Comment by orcstar** [comment]
Heya, nice idea!
Do you know if it's posible to hide the objects menu inside the room for invited people?
Thanks!!

**Reply by AlbertBaubleDeem** [reply]
Off topic, but yes it is. Check room controls.

**Comment by johnshaughnessy** [comment, marked as answer]
This is a good idea. We've tracked this request in this issue. Feel free to add more comments explaining your desired use case here: https://github.com/mozilla/hubs/issues/2424

This feature was one of several we explored during a hack week earlier this year. There's a blog post that mentions it here https://blog.mozvr.com/behind-the-scenes-at-hubs-hack-week/ and a video recording with a demo of that prototype here https://www.twitch.tv/videos/934727861?filter=all&sort=time
